packages/internal/bridge/bridge-sample-app/src/withdraw.ts

```diff
@@ -27,7 +38,141 @@ async function withdraw() {
+    console.log(approvalRes!.unsignedTx);
+
+    console.log('signing approval');
+    const approvalTxSig = await params.childWallet.signTransaction(approvalRes!.unsignedTx);
+    console.log('approvalTxSig', approvalTxSig);
+
+    const sendApprovalRes = await params.childWallet.provider.sendTransaction(approvalTxSig);
+    console.log('sendApprovalRes', sendApprovalRes);
+
+    await waitForReceipt(sendApprovalRes.hash, params.childProvider);
+  } else {
+    console.log('no approval required');
+  }
+
+  const withdrawReq: BridgeTxRequest = {
+    senderAddress: params.sender,
+    recipientAddress: params.recipient,
+    token: params.zkevmTestnetToken,
+    amount: params.amount,
+    sourceChainId: ZKEVM_TESTNET_CHAIN_ID,
+    destinationChainId: ETH_SEPOLIA_CHAIN_ID,
+    gasMultiplier: 1.1,
+  }
+
+  console.log('withdrawReq', withdrawReq)
+  let withdrawRes: BridgeTxResponse;
+  try {
+    withdrawRes = await tokenBridge.getUnsignedBridgeTx(withdrawReq);
+    console.log('withdrawRes', withdrawRes);
+  } catch(err) {
+    console.log('withdrawErr', err);
+    return
+  }
+
+  if (!withdrawRes!.unsignedTx) {
+    console.log('unable to generate withdraw tx');
+    return
+  }
+
+  console.log()
+
+  const withdrawNonce = await params.childWallet.getTransactionCount();
+  const withdrawGasPrice = await params.childProvider.getGasPrice();
+
+  withdrawRes!.unsignedTx.gasLimit = BridgeMethodsGasLimit.WITHDRAW_SOURCE;
+  withdrawRes!.unsignedTx.nonce = withdrawNonce;
+  withdrawRes!.unsignedTx.gasPrice = withdrawGasPrice.mul(2);
+
+  withdrawRes!.unsignedTx.value = ethers.BigNumber.from(withdrawRes!.unsignedTx.value);
+
+  console.log('withdrawRes.unsignedTx');
+  console.log(withdrawRes!.unsignedTx);
+
+  console.log('gasPrice', withdrawRes!.unsignedTx!.gasPrice!.toString())
+  console.log('value', withdrawRes!.unsignedTx!.value!.toString())
+
+  console.log('signing withdraw');
+  const withdrawTxSig = await params.childWallet.signTransaction(withdrawRes!.unsignedTx);
+  console.log('withdrawTxSig', withdrawTxSig);
+
+  const sendWithdrawtRes = await params.childWallet.provider.sendTransaction(withdrawTxSig);
+  console.log('sendWithdrawtRes', sendWithdrawtRes);
+
+  await waitForReceipt(sendWithdrawtRes.hash, params.childProvider);
+  
+  console.log('Withdraw submitted txHash:',sendWithdrawtRes.hash);
+
+  const txStatusReq:TxStatusRequest = {
+    sourceChainId: ZKEVM_TESTNET_CHAIN_ID,
+    transactions: [{
+      txHash: sendWithdrawtRes.hash
+    }]
+  }
+
+  let complete:boolean = false;
+  let attempts = 0;
+  while(!complete) {
+    attempts++;
+    const txStatusRes: TxStatusResponse = await tokenBridge.getTransactionStatus(txStatusReq);
+    console.log(`TxStatusResponse attempt: ${attempts}`);
+    console.log(util.inspect(txStatusRes, {showHidden: false, depth: null, colors: true}));
+    if (txStatusRes.transactions[0].status === StatusResponse.COMPLETE) {
+      complete = true;
+    }
+    await delay(10000);
```

Comment:
should this be in an else clause? feels unnecessary to wait 10s if the status is complete?